src/OrbitGl/CaptureWindow.cpp

```diff
@@ -355,7 +356,9 @@ void CaptureWindow::KeyPressed(unsigned int key_code, bool ctrl, bool shift, boo
       break;
     case 18:  // Left
```

Comment:
You can add a comment for the left / right/ down /up cases.

"Scroll horizontally or vertically if no timer is selected, otherwise, jump to the neighbour timer in that direction."

Reply:
Added